src/components/ProgressBar.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="w-full text-center">
+    <div class="mb-2 text-primary-text">{{ label }}</div>
+    <div class="overflow-hidden h-4 mb-4 text-xs flex rounded-lg bg-light-grey">
+      <div :style="`width: ${percentage}%`" class="m-progress rounded-lg bg-primary-green" />
```

Comment:
I think this should be black according to the designs:
![image](https://user-images.githubusercontent.com/25109066/114288510-854d0800-9a3e-11eb-8ad3-afc16e730b70.png)